cp1616/src/cp1616_io_controller.cpp

```diff
@@ -66,6 +83,15 @@ Cp1616IOController::~Cp1616IOController()
   
 }
 
+void operator >> (const YAML::Node &node, ControllerModuleData &module)
+{
+  node["id"] >> module.id;
```

Comment:
As long as you always use this operator within a try...catch, this should be OK.  I believe you are doing this.